resources/helm/v2.1/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -80,14 +80,3 @@ rules:
     resources: ["secrets"]
     verbs: ["get", "watch", "list"]
 
-
-  # Maistra specific
-  - apiGroups: ["maistra.io"]
-    resources: ["servicemeshmemberrolls"]
-    verbs: ["get", "list", "watch"]
-  - apiGroups: ["route.openshift.io"]
-    resources: ["routes", "routes/custom-host"]
-    verbs: ["get", "list", "watch", "create", "delete", "update"]
-  - apiGroups: ["maistra.io"]
-    resources: ["servicemeshextensions"]
-    verbs: ["get", "list", "watch"]
```

Comment:
@dgn should it also be returned from version 2.1? I'm not sure, but I suppose that it should remain.

Reply:
Yes, this change is correct.